lib/nostrum/api.ex

```diff
@@ -1355,54 +1360,82 @@ defmodule Nostrum.Api do
   end
 
   @doc """
-  Gets a user.
+  Gets a user by its `user_id`.
```

Comment:
Could we link this to the actual type in the `User` module?

Reply:
We can. Changes have been pushed to reflect this in the latest commit.